src/libasr/runtime/lfortran_intrinsics.c

```diff
@@ -591,6 +591,29 @@ LFORTRAN_API double_complex_t _lfortran_zatanh(double_complex_t x)
     return catanh(x);
 }
 
+// degrees -----------------------------------------------------------------------
+
+LFORTRAN_API float _lfortran_sdegrees(float x)
+{
+    return x*(180.0/M_PI);
+}
```

Comment:
I think we don't need this at all. This can be implemented directly in `lpython_intrinsic_numpy.py` as Python code.